dev/WindowsAppRuntime_BootstrapDLL/MddBootstrapAutoInitializer.cpp

```diff
@@ -29,6 +29,12 @@ namespace Microsoft::Windows::ApplicationModel::DynamicDependency::Bootstrap
             const HRESULT hr{ ::MddBootstrapInitialize(c_majorMinorVersion, c_versionTag, c_minVersion) };
             if (FAILED(hr))
             {
+                if (IsDebuggerPresent()) {
+                    DebugBreak();
+                }
```

Comment:
Opinion: I travel around a lot with the debugger and little is more annoying than hitting breakpoints I didn't set. I think an interested party would set a bp on `MddBootstrapInitialize` anyway, making this of no value.

Reply:
The reasoning here is that many developers are not even aware of MddBootstrapInitialize living in generated code.  So knowing to set a BP on that in case of failure is a catch-22, and the problem we're trying to solve here - drawing the developer's attention to that generated call.  Without the DebugBreak, we could pop the MessageBox unconditionally, but that's even more annoying to debug (break on all thread, find the one with a message pump, find the call frame ...).

Reply:
I just realized this is the consumer side, not the Microsoft impl side, so agree @scottj1s!